packages/nextjs/pages/builders/[0xfbCE2693FBEd73E3D464d662235DC5ADC5454C01].tsx

```diff
@@ -0,0 +1,50 @@
+import { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  const leftPanel = (
+    <div className="">
+      {/* eslint-disable @next/next/no-img-element */}
```

Comment:
Do we want [this](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) instead?

Reply:
It's not a big deal, but using NextJS `<Image>` gives you some cool optimizations. 

https://nextjs.org/docs/app/building-your-application/optimizing/images

The problem here is that you'd need to tweak `next.config.js` to allow the URL patterns of external URLs.

**One way to solve it:** What if you copy the asset to the repo (maybe create a folder inside `public`)?

You can also leave it like this. Up to you

Reply:
Leaving it as it is. Seems like the simplest solution. Not much point in tweaking config or hosting in public.